.circleci/config.yml

```diff
@@ -2,21 +2,21 @@ version: 2.1
 jobs:
   build:
     machine:
-      image: 'ubuntu-1604:201903-01'
+      image: 'ubuntu-1604:201903-01' # 'ubuntu-2004:202010-01'
     steps:
       - checkout
       - restore_cache:
-            keys: my-cache-v4
+          keys: my-cache-v4
       - run: ./install_requirements.sh
```

Comment:
whether this install_requirements script need to be changed to the new install_requirements_cpp20.sh?

Reply:
Nope. The previous builds show that changing to the new install requirement script, although compatible to all the major ubuntu versions, is extremely slow (30min). Plus there is one test failed on the CircleCI (but not on my Ubuntu 20 cloudlab machine). 

I think we will need another issue to optimize the cache and do all the right things to make sure it build fast